src/gluonts/transform/_base.py

```diff
@@ -39,11 +39,46 @@ def estimate(self, data_it: Iterator[DataEntry]) -> Iterator[DataEntry]:
         return data_it  # default is to pass through without estimation
 
     def chain(self, other: "Transformation") -> "Chain":
-        return Chain(self, other)
+        return Chain([self, other])
 
     def __add__(self, other: "Transformation") -> "Chain":
         return self.chain(other)
 
+    def __lshift__(self, dataset: Iterator[DataEntry]) -> Iterator[DataEntry]:
+        return TransformedDataset(dataset, self)
+
+    def __rrshift__(self, dataset: Iterator[DataEntry]) -> Iterator[DataEntry]:
+        return self << dataset
```

Comment:
Seems like `<<` is used only here. Do you think it could be used elsewhere? Otherwise I would avoid defining `__lshift__` at all, and just use `TransformedDataset(dataset, self)` here

Reply:
The idea was to define >> in terms of <<. 

So that you can do `data >> t` and `t << data`.